prisma/schema.prisma

```diff
@@ -132,6 +132,7 @@ model Exercise {
   id         Int            @id @default(autoincrement())
   name       String         @db.VarChar(255)
   gitHubUrl  String         @db.VarChar(255)
+  readMeUrl  String         @db.VarChar(255)
```

Comment:
This will need to be made nullable to protect the production database. Delete the `prisma/migrations/20220928163737_exercise/migration.sql` file before generating your new migration